cluster-controller/src/main/java/io/strimzi/controller/cluster/resources/TopicController.java

```diff
@@ -73,9 +76,9 @@
      * @param namespace Kubernetes/OpenShift namespace where cluster resources are going to be created
      * @param cluster   overall cluster name
      */
-    protected TopicController(String namespace, String cluster) {
+    protected TopicController(String namespace, String cluster, Labels labels) {
 
-        super(namespace, cluster);
+        super(namespace, cluster, labels.withType(TopicController.TYPE).withoutType());
```

Comment:
are you adding and removing the "type" here ?

Reply:
Excellent, thank you!